src/CarFuelMonitorEnhanced.java

```diff
@@ -10,11 +10,13 @@
  * times, i.e., the fuel level cannot drop below zero.
  * </p>
  *
- * @initially <pre>
- *     this.getCurrentFuelLevel() = 0
- * </pre>
+ * @initially
+ *
+ *            <pre>
+ *            this.getCurrentFuelLevel() = 0
+ *            </pre>
  */
-public interface CarFuelMonitorEnhanced {
+public interface CarFuelMonitorEnhanced extends CarFuelMonitorKernal {
```

Comment:
Naming convention isn't typical. 

```suggestion
public interface CarFuelMonitor extends CarFuelMonitorKernel {
```